packages/@okta/vuepress-site/docs/reference/api/admin-notifications/index.md

```diff
@@ -944,7 +944,7 @@ Refer to the [product documentation](https://help.okta.com/okta_help.htm?id=ext_
 
 | Role type                     | Label                               |
 | :---------------------------- | :---------------------------------- |
-| `API_ADMIN`                   | API Access Management Administrator |
+| `API_ADMIN` (or `API_ACCESS_MANAGEMENT_ADMIN`) | API Access Management Administrator |
```

Comment:
```suggestion
| `API_ADMIN` or `API_ACCESS_MANAGEMENT_ADMIN` | API Access Management Administrator |
```

Reply:
Addressed in https://github.com/okta/okta-developer-docs/commit/d9c59f41591f3b39c75ecb0e2354db187cb87a99